karma.conf.babel.js

```diff
@@ -54,9 +54,7 @@ export default (config) => {
         ]
       },
       resolve: {
-        modulesDirectories: [
-          'node_modules',
-        ],
+        root: paths.root,
```

Comment:
Removed `modulesDirectories`, it was set to the default value.  Added `root` so we can require from root in tests.